presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java

```diff
@@ -444,6 +445,19 @@ public FeaturesConfig setOptimizeTopNRowNumber(boolean optimizeTopNRowNumber)
         return this;
     }
 
+    public boolean isOptimizeTopNRanking()
```

Comment:
You should remove `optimizeTopNRowNumber` feature config too (and add it to `@DefunctConfig` list)

Reply:
Can't move to DefunctConfig list quite yet. We probably need a minimal transition period for people to be warned that the old config is being deprecated in the future.

Reply:
`isOptimizeTopNRowNumber` is not used after this commit, so you effectively defunc it.